src/pilot/deploy-overcloud.py

```diff
@@ -603,7 +603,7 @@ def main():
         env_opts += " -e ~/pilot/templates/overcloud/environments/" \
                     "network-isolation.yaml" \
                     " -e ~/pilot/templates/network-environment.yaml" \
-                    " -e /usr/share/openstack-tripleo-heat-templates/environments/services/neutron-ovs.yaml" \
+                    " -e /home/stack/pilot/templates/overcloud/environments/services/neutron-ovs.yaml" \
```

Comment:
Why don't we do ~/pilot instead of /home/stack/pilot?

Reply:
Good catch. Fixed it